### PR TITLE
[Merged by Bors] - chore: fix upper/lowercase in comments

### PR DESCRIPTION
**Pull request by urkud**
* Run a non-interactive version of `fix-comments.py` on all files.
* Go through the diff and manually add/discard/edit chunks.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Filtered.lean [hunk 1]
```diff
@@ -566,8 +566,8 @@ noncomputable def minToRight (j j' : C) : min j j' ⟶ j' :=
   (IsCofilteredOrEmpty.cone_objs j j').choose_spec.choose_spec.choose
 #align category_theory.is_cofiltered.min_to_right CategoryTheory.IsCofiltered.minToRight
 
-/-- `eq f f'`, for morphisms `f f' : j ⟶ j'`, is an arbitrary choice of object
```
Comment (Ruben-VandeVelde) [inline Mathlib/CategoryTheory/Filtered.lean:569]: I think these are about `eq` below

File: Mathlib/Data/Int/Bitwise.lean [hunk 2]
```diff
@@ -94,7 +94,7 @@ theorem bodd_mul (m n : ℤ) : bodd (m * n) = (bodd m && bodd n) := by
   simp only [negSucc_coe, bodd_neg, bodd_coe, ←Nat.bodd_mul]
 -- Porting note: Heavily refactored proof, used to be:
 -- `by cases m with m m; cases n with n n;`
--- `simp [← int.mul_def, int.mul, -of_nat_eq_coe, bool.bxor_comm]`
+-- `simp [← Int.mul_def, int.mul, -of_nat_eq_coe, Bool.xor_comm]`
```
Comment (Ruben-VandeVelde) [inline Mathlib/Data/Int/Bitwise.lean:97]: The other two may be off too

File: Mathlib/GroupTheory/MonoidLocalization.lean [hunk 3]
```diff
@@ -61,7 +61,7 @@ this structure.
 To reason about the localization as a quotient type, use `mk_eq_monoidOf_mk'` and associated
 lemmas. These show the quotient map `mk : M → S → Localization S` equals the
 surjection `LocalizationMap.mk'` induced by the map
-`monoid_of : localization_map S (localization S)` (where `of` establishes the
+`monoid_of : localization_map S (Localization S)` (where `of` establishes the
```
Comment (Ruben-VandeVelde) [inline Mathlib/GroupTheory/MonoidLocalization.lean:64]: How about `localization_map` and `monoid_of`?

**Comment by semorrison** [comment]
I skimmed through to Mathlib/RingTheory/Ideal/Over.lean, after which github wasn't showing me the diffs without clicking once for each file. I'd be happy to merge. If anyone wants to look through the tail, that's fine too.

File: Mathlib/Logic/Equiv/Defs.lean [hunk 4]
```diff
@@ -943,7 +943,7 @@ protected def congr {ra : α → α → Prop} {rb : β → β → Prop} (e : α
 #align quot.congr_mk Quot.congr_mk
 
 /-- Quotients are congruent on equivalences under equality of their relation.
-An alternative is just to use rewriting with `eq`, but then computational proofs get stuck. -/
+An alternative is just to use rewriting with `Eq`, but then computational proofs get stuck. -/
```
Comment (Ruben-VandeVelde) [inline Mathlib/Logic/Equiv/Defs.lean:946]: Nope

File: Mathlib/Logic/Equiv/Defs.lean [hunk 5]
```diff
@@ -972,7 +972,7 @@ protected def congr {ra : Setoid α} {rb : Setoid β} (e : α ≃ β)
 #align quotient.congr_mk Quotient.congr_mk
 
 /-- Quotients are congruent on equivalences under equality of their relation.
-An alternative is just to use rewriting with `eq`, but then computational proofs get stuck. -/
+An alternative is just to use rewriting with `Eq`, but then computational proofs get stuck. -/
```
Comment (Ruben-VandeVelde) [inline Mathlib/Logic/Equiv/Defs.lean:975]: Nope

File: Mathlib/Tactic/RestateAxiom.lean [hunk 6]
```diff
@@ -18,7 +18,7 @@ It is still present for backwards compatibility but will probably be removed in
 # Original Docstring
 
 `restate_axiom` makes a new copy of a structure field, first definitionally simplifying the type.
-This is useful to remove `auto_param` or `opt_param` from the statement.
+This is useful to remove `autoParamₓ` or `optParam` from the statement.
```
Comment (Ruben-VandeVelde) [inline Mathlib/Tactic/RestateAxiom.lean:21]:
```suggestion
This is useful to remove `autoParam` or `optParam` from the statement.
```

File: Mathlib/Data/Int/Bitwise.lean (hunk 2 above)
Comment (urkud) [reply Mathlib/Data/Int/Bitwise.lean:97]: This is a quoted Lean 3 proof, so I'm going to revert this chunk.

File: Mathlib/GroupTheory/MonoidLocalization.lean (hunk 3 above)
Comment (urkud) [reply Mathlib/GroupTheory/MonoidLocalization.lean]: @Ruben-VandeVelde , could you please have another look at this file?

**Review by Ruben-VandeVelde — APPROVED** [review]
Thanks! Just the one left.

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**